packages/Text/Text.md

```diff
@@ -1,14 +1,25 @@
+### Minimal usage
+
 ```jsx
 <Text>Tell us what you think</Text>
 ```
 
-By default, the Text component will inherit font properties from its parent. Use the props to override this behaviour.
+### Usage criteria
+
+- Only used to modify text in `Paragraph`.
```

Comment:
Let's rephrase this to say 'Should only be used to modify text in `Paragraph` as there are exceptions such as PriceLockup, tertiaryText in ExpandCollapse, and other standoffish text.